src/foam/u2/PropertyBorder.js

```diff
@@ -145,7 +145,10 @@
         show(visibilitySlot).
         add(this.slot(function(reserveLabelSpace, prop$label){
           let el = this.E().addClasses([this.myClass('label'), 'p-semiBold']);
-          return prop$label ? el.add(prop.label$) : ( reserveLabelSpace ? el : undefined )
+          return prop$label ? el.callIfElse(prop.labelFormatter,
```

Comment:
I already merged, but could you also pass the labelFormatter the prop and then have the default perform the old behaviour? Then the code would be simpler because you would just always use the labelFormatter.